CMPT276-SFU-RSC/Meeting-Application
# Feature/dashboard

Created the endpoint /dashboard which pulls polls from the data base that have the same user ID as the session. (note that the ID is set to 1 hardcoded, but can be fixed by uncommenting all the code to do with sessions).

/Dashboard will be the "main" page for the application, where others branch from

Testing:
- Make sure it can view several polls/no polls from database
- Make sure clicking "View Event" takes the user to the correct end point to view that poll (endpoint is not yet implemented)
- Make sure it cant see polls it has no permissions to view
- Verify sessions work (once uncommented from PollController.java)
